src/lib/tpm.c

```diff
@@ -486,6 +499,77 @@ bool files_load_bytes_from_path(const char *path, UINT8 *buf, UINT16 *size) {
 LOAD_TYPE(TPM2B_PUBLIC, public)
 LOAD_TYPE(TPM2B_PRIVATE, private)
 
+CK_RV tpm_get_token_info_from_tpm (tpm_ctx *ctx, CK_TOKEN_INFO *info) {
+
+    uint32_t revision;
+    uint32_t version;
+    uint32_t manufacturer;
+    unsigned char manufacturerID[5] = {0}; // 4 bytes + '\0'
+    uint32_t vendor[4];
```

Comment:
Anyway to remove the magic buffer size here?

Reply:
sizeof(UINT32)+1